tools/app/cypress/e2e/app.cy.ts

```diff
@@ -10,76 +10,137 @@ describe('Navigation', () => {
   beforeEach(() => {
     cy.visit('http://localhost:3000');
   });
+  it('Create new base module', () => {
+    // Creates a new base module with name Basic Acceptance Test
+    cy.exec('cd ../../../&&git clone git@github.com:CyberismoCom/module-base.git&&cyberismo create project "Basic Acceptance Test" bat cyberismo-bat');
+    cy.wait(1000); // short wait to avoid random failures
+    cy.exec('cd ../../../cyberismo-bat&&cyberismo import module ../module-base&&cyberismo create card base/templates/page');
+    cy.get('h4').contains('Basic Acceptance Test'); // Verify project name
+    cy.get('p').contains('Please select a card'); // Verify text on cards page
```

Comment:
I think the tests should clean up after themselves and delete the cyberismo-bat folder. Currently running tests twice fails because the folder already exists.